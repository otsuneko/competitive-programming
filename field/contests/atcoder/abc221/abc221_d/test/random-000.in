6
2 4
10 10
9 3
8 4
2 7
7 4
