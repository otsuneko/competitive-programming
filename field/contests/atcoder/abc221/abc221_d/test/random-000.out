9 5 3 1 0 0
